### PR TITLE
WIP: Skip download_repos on Staging-Images runs

### PR DESCRIPTION
**Pull request by grisu48**
On Staged image test runs the step of downloading maintenance updates should be skipped.

- Related ticket: https://progress.opensuse.org/issues/94144
- Verification run: http://duck-norris.qam.suse.de/tests/6619 (cancelled after successful setup)

### CODE REVIEW

File: tests/publiccloud/download_repos.pm
Comment (pdostal) [inline tests/publiccloud/download_repos.pm:97]: You can move this `cd` to the `else` condition one line above too.

File: tests/virtualization/universal/patch_and_reboot.pm
Comment (pdostal) [inline tests/virtualization/universal/patch_and_reboot.pm:43]: You probably know this does not belong here 😄

**Review by pdostal — APPROVED** [review]
I think this is nice 😄 Just please resolve those two small comments before merging.

File: tests/publiccloud/download_repos.pm
Comment (grisu48) [reply tests/publiccloud/download_repos.pm:97]: That was kept on purpose there, so that after the test run the VM will be in `/root`, regardless if updates have been downloaded or not.

File: tests/virtualization/universal/patch_and_reboot.pm
Comment (grisu48) [reply tests/virtualization/universal/patch_and_reboot.pm:43]: I see what you mean :slightly_smiling_face: This commit doesn't belong there.

File: tests/publiccloud/download_repos.pm
Comment (pdostal) [reply tests/publiccloud/download_repos.pm:97]: ok 😄 I've no hard feeling about it.

File: tests/virtualization/universal/patch_and_reboot.pm
Comment (pdostal) [reply tests/virtualization/universal/patch_and_reboot.pm:43]: Please fix it and close this thread then ;)

**Review by b10n1k — APPROVED** [review]
+1

**Review by jlausuch — CHANGES_REQUESTED** [review]
LGTM but will merge after commit removal :)

**Review by jlausuch — APPROVED** [review]

**Comment by grisu48** [comment]
This PR is obsolete. The issue has been solved by https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12763 in the meantime. Closing it.

**Pull request closed without merging**
